.gitignore

```diff
@@ -13,8 +13,13 @@ Gemfile.lock
 vendor/
 **/velocity.log
 
-# temp directory is ignored so that it can be used to store temporary files such as local testing configurations.
+# temp & local directories are ignored so that they can be used to store temporary files such as local testing configurations.
 temp/
+local/
```

Comment:
Is there a way not to generate these files in the project root? Put them into local directory instead? 

This is one of the reasons I put all my custom config files in the temp directory.

Reply:
That would be much better. The only problem is that user will have to set the working directory to `temp/` before running the server :( But hey, maybe there's a way to somehow prefix all file paths instead of writing to the root of the current working dir. I'll explore that possibility.